addons/website_slides/controllers/mail.py

```diff
@@ -22,21 +22,25 @@ def portal_chatter_post(self, res_model, res_id, message, **kw):
             if rating_value and slide_channel and request.env.user.partner_id.id == int(kw.get('pid')):
                 # apply karma gain rule only once
                 request.env.user.add_karma(slide_channel.karma_gen_channel_rank)
+            result.update({
+                'default_rating_value': rating_value,
+                'rating_avg': slide_channel.rating_avg,
+                'rating_total': slide_channel.rating_count,
```

Comment:
Any reason you're using a different name here? (rating_total instead of rating_count)

Rating total makes it seem like it's the sum of all ratings, when it's actually the number of ratings, which rating_count makes pretty clear.

Reply:
> Any reason you're using a different name here? (rating_total instead of rating_count)
> 
> Rating total makes it seem like it's the sum of all ratings, when it's actually the number of ratings, which rating_count makes pretty clear.

the name of rating_total already uses to everywhere in the widget and XML part as well, so that's why we have used the same name (rating_total) here also. if we use rating_count to here then we need to change everywhere. for minimal diff we have used rating_total.
see: https://github.com/odoo/odoo/pull/47746/files#diff-5dd8343a98ebea1c3fae4984b1a8ba1fL29
https://github.com/odoo/odoo/pull/47746/files#diff-e5cf002f35cb85969f62891bd9f03e71L38
...
...

now for that, we will be using rating_count instead of rating_total to everywhere for more code readability and pretty clear.